test/mosaic-integration-tests/pom.xml

```diff
@@ -53,6 +53,23 @@
             <version>${mosaic.version}</version>
             <scope>test</scope>
         </dependency>
+        <!--For applications-->
+        <dependency>
+            <groupId>org.eclipse.mosaic</groupId>
+            <artifactId>mosaic-rti-core</artifactId>
+            <version>${mosaic.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.mosaic</groupId>
+            <artifactId>mosaic-application</artifactId>
+            <version>${mosaic.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.mosaic</groupId>
+            <artifactId>mosaic-objects</artifactId>
+            <version>${mosaic.version}</version>
+        </dependency>
+
     </dependencies>
 
     <build>
```

Comment:
Our mosaic-extended project requires the Send/Receive applications as well for some integration tests. Those are currently failing as you moved them to eclipse-mosaic. We need to make them available for those integration tests as well. This is how you can achieve this:

1) Move all apps into the package `org.eclipse.mosaic.test.app....` (you still can use additionals subpackages)
2) add an `<include>org/eclipse/mosaic/test/app/**</include>` to the `create-test-jar-with-rules` section below
3) For the legacy integration tests in mosaic-extended, you have to copy the `target/mosaic-integration-tests-20.1-SNAPSHOT-tests.jar` to `mosaic/lib/mosaic` by adjusting the `setup_testing.sh`.

Reply:
Am I right in assuming, that the `mapping_config.json`'s have to be adjusted to `org.eclipse.mosaic.test.app...`

Reply:
Correct